_docs/admin/setup/mail/mail-routing.md

```diff
@@ -15,8 +15,28 @@ Navigate to **Setup >> Mail >> Incoming Mail >> Mail Routing**.
 
 You can configure rules to automatically route new mail to groups based on message properties.
 
-For example, you may want to route messages addressed to `support@*` to the **Support** group, `orders@*` to **Sales**, and `receipts@*` to **Billing**.
+For example, you may want to route messages addressed to `support@*` to the **Support** group, `orders@*` to **Sales**, and `receipts@*` to **Billing**. Adding an <b>*</b> after the **@** symbol makes the rule a wildcard that link multiple domains to that mailbox. 
 
 These messages will be delivered to the **Inbox** bucket in those groups, and group managers can configure additional sorting from there.
 
+<h3>Sticky Rules vs Managed Rules</h3>
```

Comment:
Use `###` in Markdown syntax and not HTML `<h3>` for the headings here. You shouldn't be writing any HTML except for screenshots and within the note/warning blocks.